modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionCommandListener.java

```diff
@@ -90,38 +123,150 @@ else if (clo.command() instanceof ReplaceCommand) {
                     clo.success(false);
```

Comment:
```
                if ((current == null && !expected.hasValue()) ||
                    equalValues(current, expected)) {
                    storage.put(key, cmd.getRow());
                    clo.success(true);
                }
```
there is possible NPE because `current` might be null when you pass it to `equalValues`

Reply:
The equalValues is no longer throw NPE when one row is null.